### PR TITLE
PowerPC64 fixes

### PR DESCRIPTION
**Pull request by antonblanchard**
A number of fixes for PowerPC64

### CODE REVIEW

File: src/codegen.cpp [hunk 1]
```diff
@@ -5797,7 +5797,7 @@ extern "C" void jl_init_codegen(void)
             targetFeatures);
     assert(jl_TargetMachine && "Failed to select target machine -"
                                " Is the LLVM backend for this CPU enabled?");
-#if defined(USE_MCJIT) && !defined(_CPU_ARM_)
+#if defined(USE_MCJIT) && (!defined(_CPU_ARM_) && !defined(_CPU_PPC64_))
```
Comment (yuyichao) [inline src/codegen.cpp]: I'm wonder what's wrong about fastisel... it works on AArch64 but not AArch32 for example......

What's the issue you see with FastISel?

Comment (antonblanchard) [inline src/codegen.cpp]: Most likely a bug in the PowerPC64 fast-isel code, it doesn't get a whole lot of testing. We are working to find the bugs in llvm and fix them.

Comment (yuyichao) [inline src/codegen.cpp]: So it's a different kind of failure with ARM backend, where the codegen messes up addressing mode and causes segfault...

**Comment by Keno** [comment]
LGTM

**Pull request merged**